packages/scene-composer/src/utils/componentSettingsUtils.spec.ts

```diff
@@ -11,6 +11,12 @@ describe('componentSettingsUtils', () => {
       expect(tagSettings).toEqual(DEFAULT_TAG_GLOBAL_SETTINGS);
     });
 
+    it('should return default overlay settings', () => {
+      const state = { getSceneProperty: jest.fn() } as any;
```

Comment:
no any here - that way if the expected input changes thru a refactor, this can be discovered via typescript error

Reply:
oops, that's why the warning count is larger than my early update lol.. I'll fix it